### ISSUE TITLE
SCSS Property does not exist on type '{}'

### ISSUE THREAD

**Issue by jerryc05** [mrmckeb/typescript-plugin-css-modules#188]
1. enable `"goToDefinition": true` in `tsconfig.json`

2. create a sample `index.scss` file look like this:
```scss
.classA {
  height: 100%;
  &>.classB {
    width: 100%;
  }
}
```

3. in a typescript file, import the scss as `import css from './index.scss'` 

4. after import, only `css.classA` is valid. accessing `css.classB` gives 
```
Property 'classB' does not exist on type 'typeof import("???/index.scss")'
```

5. However, turning `goToDefinition` off makes `classB` valid, but I would like to keep it on and still be able to use `css.classB`

**Comment by jerryc05** [comment]
version 4.1.1

**Comment by mrmckeb** [comment]
Thanks for raising this. I'll take a look.

The `goToDefinition` is new, and a little primitive right now. At a guess I'd say that this is because the class wasn't matched, and I didn't handle that properly. Sorry that I didn't spot this during testing.

I'm currently half-way through a big refactor, and will aim to take a look straight after that, as the workaround should be OK for the moment.

**Comment by mrmckeb** [comment]
From my testing, this seems to have been resolved by #201. I'll release tomorrow as v4.2.0.

**Comment by mrmckeb** [comment]
This should now be resolved in `v4.2.1`:
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.1
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.0
